kubespawner/spawner.py

```diff
@@ -355,6 +355,24 @@ def _namespace_default(self):
         """
     )
 
+    pod_ip_template = Unicode(
```

Comment:
Or, `notenook_ip_template`?

Reply:
pod_ip_template seems good, to match other traitlet names here.

Reply:
Thank you for your work on this @dtaniwaki! Could you also try to:
- communicate that this is how KubeSpawner will try to access the pod after creation (running in the hub pod), rather than what IP the pod will be created with
- communicate that this is optional, and we otherwise default to accessing the pod using its by k8s automatically assigned IP

@yuvipanda and @dtaniwaki I also suggest we name this `pod_connect_ip_template`, it would mimic the naming of [c.JupyterHub.hub_connect_ip](https://jupyterhub.readthedocs.io/en/stable/api/app.html#jupyterhub.app.JupyterHub.hub_connect_ip). What do you think?

Reply:
@yuvipanda Which do you think better?

Reply:
@dtaniwaki I think we can just call it `pod_connect_ip`, and do template expansion, since the template expansion is the same thing we do for many other properties here.